### DISCUSSION TITLE
Quick screenshot UI

### DISCUSSION THREAD

**Discussion by redtide** [lxqt/lxqt#2094] [Q&A] [answered]
I recall there was a simple screenshot grabber, maybe was in LXDE, similar (if not same) as the one contained in LXImage-Qt.
Now I tried also Screengrab and I would like (or not found how) to not get a screenshot when I run the application, just like gnome-screenshot. LXImage-Qt one is closer to this, except it opens the image instead to ask where to save it, clipboard or file.
Is there a setting in Screengrab/LXImage-Qt to achieve this or should be a new application?

**Comment by tsujan** [comment, marked as answer]
I don't understand why a screenshot utility shouldn't take a screenshot when it appears. That would be useless even as an option.

As for lximage-qt, it's an image viewer. We wanted to remove its screenshot functionality but, since it was used by many, we left it as it was. It'll receive no change.

**Reply by stefonarch** [reply]
If "minimize to tray" is set if behaves mostly like that, except for the first one.

**Reply by redtide** [reply]
> I don't understand why a screenshot utility shouldn't take a screenshot when it appears. That would be useless even as an option.

because it doesn't know what kind of screenshot I'm going to make, if for the focused window, the desktop or just a selected area.
So I guess I can make one by extracting and adapting the one in LX-Image. Thank you for answers.

**Reply by tsujan** [reply]
Screengrab starts with the user's last choice.

**Comment by redtide** [comment]
I forgot to notify about the app mentioned, which is [here](https://github.com/redtide/sqreenshot/), it uses some code from LXImage-Qt (screenshotdialog.cpp -> dialogscreenshot.cpp), hope it's OK with the license.
